src/App.jsx

```diff
@@ -29,9 +29,14 @@ const levelClearedMsg = [
   "🎉 Congratulations, you've conquered!",
 ];
 
+
 function App() {
-  const [pieces, setPices] = useState([]);
-  const [level, setLevel] = useState(1);
+   
+
+  const currentGameLevel = parseInt(localStorage.gameLevel);
+  
+  const [pieces, setPieces] = useState([]);
```

Comment:
avoid accessing localStore value directly, it causes issues in different browsers.
Use, `localStorage.getItem("gameLevel")`